### PR TITLE
Fix Entity dropdown using "multiple" option

### PR DESCRIPTION
**Pull request by Aeto-J**
Added parenthesis to fix error which occurs when creating an Entity multiple choice dropdown: "Uncaught Exception TypeError: array_combine(): Argument 2 ($values) must be of type array, null given in glpi10.0.3/src/Html.php at line 4857"

Added Dropdown::testShow to unit test multiple choice dropdown creation on Entity's itemtype

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

**Review by tomolimo — APPROVED** [review]

File: tests/functionnal/Dropdown.php [hunk 1]
```diff
@@ -1248,6 +1248,188 @@ public function testGetDropdownUsers($params, $expected)
         $this->array($result)->isIdenticalTo($expected);
     }
 
+    /**
+     * Summary of testShow
+     */
+    public function testShow()
```
Comment (cedric-anne) [inline tests/functionnal/Dropdown.php:1254]: This kind of test is not maintainable. Not sure it has sense to add it.

@trasher @orthagh your opinion ?

Comment (orthagh) [reply tests/functionnal/Dropdown.php:1254]: Agree.

If a test should be done on html output, i guess i'll go with a dom crawler (like `symfony/dom-crawler` we already have) to check presence of some tags (`select[multiple]` for example).
But doing an equality of all generated html seems a bad idea

Comment (trasher) [reply tests/functionnal/Dropdown.php:1254]: I agree with @orthagh; the test cans be kept, but not on an entire html output

File: tests/functionnal/Dropdown.php [hunk 2]
```diff
@@ -1248,6 +1248,188 @@ public function testGetDropdownUsers($params, $expected)
...
+            'display' => false,
+            'multiple' => true,
+            'value' => [0, 1],
+            'rand' => "123456"
+        ]);
+        $result = preg_replace('/^_idor_token: .*,$/m', '_idor_token: "8961133e2dec757572a4aeed81419b585e9cead899f1c6bea9e344555cfa89a2",', $result);
+        $result = preg_replace(['/myModal[0-9]+/', '/myModalEl[0-9]+/'], ['myModal123456','myModalEl123456' ], $result);
+        $this->string($result)->isIdenticalTo($expected);
+    }
+
```
Comment (cedric-anne) [inline tests/functionnal/Dropdown.php:1432]:
```suggestion
```

File: src/Dropdown.php [hunk 3]
```diff
@@ -155,8 +155,8 @@ public static function show($itemtype, $options = [])
             $name = $params['toadd'][$params['value']];
         } else if (
             !$params['multiple']
-            && $params['value'] > 0
-            || ($itemtype == "Entity" && $params['value'] >= 0)
+            && ($params['value'] > 0
+            || ($itemtype == "Entity" && $params['value'] >= 0))
```
Comment (cedric-anne) [inline src/Dropdown.php:159]:
```suggestion
            && ($params['value'] > 0 || ($itemtype == "Entity" && $params['value'] >= 0))
```

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Comment by Aeto-J** [comment]
Hello, 
Thank you for the fix 
best regards

**Pull request merged**